addons/mail/models/res_partner.py

```diff
@@ -78,8 +78,13 @@ def _message_get_default_recipients(self):
 
     @api.model
     @api.returns('self', lambda value: value.id)
-    def find_or_create(self, email, assert_valid_email=False):
-        """ Override to use the email_normalized field. """
+    def find_or_create(self, email, assert_valid_email=False, default_create_values_getter=None):
```

Comment:
There is now a "batch" method introduced for mail template :) it is a bit more complex as it is batch-enabled, but simply retarget the code to the new batch method (that already accepts additional values I think).

Reply:
It’s done (but still using the function in parameter as we don’t have the record anymore in that method).